README.md

```diff
@@ -9,6 +9,11 @@
 <h1>ClickHouse Java Client & JDBC Driver</h1>
 </div>
 
+
+> [!CAUTION]
+> - We are planning to deprecate client-v1 (projects: `clickhouse-client`, `clickhouse-http-client`, `clickhouse-data`) in *4 Monthes*. It means no support and new releases
```

Comment:
1. Let's use the absolute dates. We won't update the README file every month to decrease the conter
2. We need to provide an alternative solution: `client-v2`
3. `It means no support and new releases` --> `It means no support and new releases for the deprecated version after DD.MM.YYYY`